components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -2848,7 +2848,7 @@ subroutine prevent_ice_overdepletion(pres,t_atm,qv,latent_heat_sublim,inv_dt,
    real(rtype) :: dumqv_sat_i, qdep_satadj
 
    dumqv_sat_i = qv_sat(t_atm,pres,1)
-   qdep_satadj = (qv-dumqv_sat_i)/(1._rtype+bfb_square(latent_heat_sublim)*dumqv_sat_i/(cp*rv*bfb_square(t_atm)))*inv_dt
+   qdep_satadj = (qv-dumqv_sat_i)/(1._rtype + bfb_square(latent_heat_sublim)*dumqv_sat_i/(cp*rv* bfb_square(t_atm) ))*inv_dt
```

Comment:
The extra spaces you added here shouldn't matter, right? I like the new spacing better, but I don't think this is really relevant, right?

Reply:
This is mistry to me when I added the space, this code can be built successfully, otherwise, it was failed. more specifically, it is the "+" operator that needs an additional space.

Reply:
Where the error says,
PGF90-S-0038-Symbol, bfb_square, has not been explicitly declared (/autofs/nccs-svm1_home1/yuanx/scream_ccn_fix/components/cam/src/physics/cam/micro_p3.F90)
  0 inform,   0 warnings,   1 severes, 0 fatal for prevent_ice_overdepletion
gmake[2]: *** [cmake/atm/CMakeFiles/atm.dir/__/__/cam/src/physics/cam/micro_p3.F90.o] Error 2
gmake[2]: Leaving directory `/gpfs/alpine/cli115/proj-shared/yuanx/e3sm_scratch/ERS.ne4pg2_ne4pg2.F2010-SCREAM-LR.summit_pgi.20201012_113415_x0cbbb/bld/cmake-bld'
gmake[1]: *** [cmake/atm/CMakeFiles/atm.dir/all] Error 2
gmake[1]: Leaving directory `/gpfs/alpine/cli115/proj-shared/yuanx/e3sm_scratch/ERS.ne4pg2_ne4pg2.F2010-SCREAM-LR.summit_pgi.20201012_113415_x0cbbb/bld/cmake-bld'
gmake: *** [all] Error 2

Reply:
Ok, this makes no sense to me but since the code is working now and having spaces before/after the '+' operator looks more pleasant anyways, I say we just go with it.